src/scripts/settings/userSettings.js

```diff
@@ -230,6 +230,19 @@ export class UserSettings {
         return toBoolean(this.get('enableThemeSongs', false), false);
     }
 
+    /**
+     * Get or set 'Theme Songs' volume level.
+     * @param {number|undefined} [val] - Volume level to set.
+     * @return {number} Current volume level.
+     */
+    themeSongsVolumeLevel(val) {
+        if (val !== undefined) {
+            return this.set('themeSongsVolumeLevel', val.toString());
+        }
+
+        return parseInt(this.get('themeSongsVolumeLevel') || '100', 10);
```

Comment:
IMO, it would be better to make the default volume lower.
```suggestion
        return parseInt(this.get('themeSongsVolumeLevel') || '50', 10);
```